requirements.txt

```diff
@@ -1,7 +1,7 @@
 pyyaml>=3.0,<=3.99
 click>=6.0,<=6.99
 funcsigs==1.0.2 ; python_version == '2.7'
-future>-0.16,<=0.99 ; python_version == '2.7'
+future>=0.16,<=0.99 ; python_version == '2.7'
```

Comment:
I specifically did not want to parse here. They have different uses as linked in the PR. If someone wants to pin requirements.txt to versions for its intended use. that's fine, but probably a different PR.